docs/pages/Kratos/Processes/Calculation/calculate_nodal_distance_to_skin_process.md

```diff
@@ -72,5 +85,8 @@ Defines the skin model part for distance computations. Like the volume model par
 ##### `distance_database`
 This flag indicating whether historical variable is enabled or otherwise non-historical is considered. It is set by string, options are `nodal_historical` or `nodal_non_historical`. Default value is `nodal_historical`.
 
+#### `save_distance_in_skin`
+This flag indicates if the distance is going to be saved in the skin. By default is false, if activated will save the maximum distance in the found skin geometry and set the flag `VISITED`. This also deactivate shared memory parallelization.
```

Comment:
save_max_distance_in_skin would be more descriptive

Reply:
Done